neuralforecast/common/_base_multivariate.py

```diff
@@ -151,7 +160,12 @@ def on_fit_start(self):
         random.seed(self.random_seed)
 
     def configure_optimizers(self):
-        optimizer = torch.optim.Adam(self.parameters(), lr=self.learning_rate)
+        if self.optimizer:
+            optimizer = self.optimizer(
+                params=self.parameters(), **self.optimizer_kwargs
+            )
+        else:
+            optimizer = torch.optim.Adam(self.parameters(), lr=self.learning_rate)
```

Comment:
since `learning_rate` is a common argument across models, it might be better to catch if `self.optimizer` receives `lr` and update `self.optimizer_kwargs` to prevent confusion regarding the usage of this parameter in case a custom optimizer is used. for example, 

```python
optimizer_signature = inspect.signature(self.optimizer)
optimizer_kwargs = deepcopy(self.optimizer_kwargs)
if "lr" in optimizer_signature:
    if "lr" in optimizer_kwargs:
          warnings.warn("ignoring learning rate passed in optimizer_kwargs, using the model's learning rate")
     optimizer_kwargs["lr"] = self.learning_rate
```